css/components.css

```diff
@@ -0,0 +1,517 @@
+/* button Started */
+.avtr,.avtr-clr,.sm-avtr-clr,.avtr-sq,.sm-avtr-sq{
+    position: relative;
+    padding: 4px;
+}
+.avtr-clr > h4,.sm-avtr-clr > h4,.avtr-sq> h4{
+    border-radius: 50%;
+    background: var(--primary-light);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 2.5rem;
+    height: 2.5rem;
+    color: var(--inner-text);
+}
+.avtr-sq>h4{
+    border-radius: 0.25rem;
+}
+.sm-avtr-clr > h4{
+    width: 2rem;
+    height: 2rem;
+}
+.avtr-clr > img , .sm-avtr-clr > img , .avtr-sq > img,.sm-avtr-sq > img {
+    object-fit: cover;
+    border-radius: 50%;
+}
+
+.avtr-sq > img{
+    border-radius: 0.375rem;
+}
+.sm-avtr-sq > img{
+    border-radius: 0.25rem;
+}
+.avtr-clr > img, .avtr-sq > img {
+    width: 2.5rem;
+    height: 2.5rem;
+}
+.sm-avtr-clr > img, .sm-avtr-sq > img{
+    width: 32px;
+    height: 32px;
+}
+.bdge{
+    box-sizing: border-box;
```

Comment:
#### Avoid Repetition, you have already declared `box-sizing` in your `bucket.css`